# HG changeset patch
# User Rik <rik@octave.org>
# Date 1484351494 28800
#      Fri Jan 13 15:51:34 2017 -0800
# Node ID 52d06f98c15e139f092517ebabc10659b7ab0c7e
# Parent  a11950e51b7f1dbbe73dbf92585d374117e8c3e4
Fix typo in typedef for octave_execution_exception (bug #50027)

* liboctave/cruft/misc/quit.h: Use typedef to make octave_execution_exception an alias for octave::execution_exception, NOT, octave::exit_exception.

diff --git a/liboctave/cruft/misc/quit.h b/liboctave/cruft/misc/quit.h
--- a/liboctave/cruft/misc/quit.h
+++ b/liboctave/cruft/misc/quit.h
@@ -141,17 +141,17 @@ namespace octave
 
   class
   interrupt_exception
   {
   };
 }
 
 OCTAVE_DEPRECATED ("use 'octave::execution_exception' instead")
-typedef octave::exit_exception octave_execution_exception;
+typedef octave::execution_exception octave_execution_exception;
 
 OCTAVE_DEPRECATED ("use 'octave::exit_exception' instead")
 typedef octave::exit_exception octave_exit_exception;
 
 OCTAVE_DEPRECATED ("use 'octave::interrupt_exception' instead")
 typedef octave::interrupt_exception octave_interrupt_exception;
 
 #endif
